nerfstudio/models/nerfacto.py

```diff
@@ -281,6 +334,12 @@ def get_image_metrics_and_images(
 
         images_dict = {"img": combined_rgb, "accumulation": combined_acc, "depth": combined_depth}
 
+        # normals to RGB for visualization. TODO: use a colormap
+        if "normals" in outputs:
+            images_dict["normals"] = outputs["normals"]
```

Comment:
Does this not need to be scaled into the range [0, 1]? Normals would be in the range `[-1, -1]`.